code/datums/martial/wrestling.dm

```diff
@@ -354,7 +359,7 @@
 					"<span class='userdanger'>You're roundhouse-kicked by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
 	to_chat(A, "<span class='danger'>You roundhouse-kick [D]!</span>")
 	playsound(A.loc, "swing_hit", 50, TRUE)
-	D.adjustBruteLoss(rand(10,20))
+	D.adjustBruteLoss(rand(2,5))
```

Comment:
B R U H

Why would I EVER use this move? It's worse than just punching someone on harm intent (and flipping/screaming using a hotkey).